### DISCUSSION TITLE
Acconto Google per login

### DISCUSSION THREAD

**Discussion by Davide031** [iisgiua/giuaschool#733] [Richieste di aiuto] [answered]
Buonasera sarebbe possibile inserire la possibilità di accedere con Google (gmail.com come dominio)o se è già possibile come posso fare?
Grazie

**Comment by trinko** [comment, marked as answer]
L'uso degli utenti Google funziona solo se si attiva la piattaforma Google Workspace, che è gratuita  per le scuole, nella versione base. 
E' indispensabile usare il dominio istituzionale (nomescuola.edu.it), per essere riconosciuti come scuole.

Vedi: [https://support.google.com/a/answer/2856827?hl=it](https://support.google.com/a/answer/2856827?hl=it)
